sdk/core/Azure.Core/src/TokenCredential.cs

```diff
@@ -11,6 +12,13 @@ namespace Azure.Core
     /// </summary>
     public abstract class TokenCredential
     {
+        /// <summary>
+        /// Returns a <see cref="TimeSpan"/> value representing the amount of time to subtract from the token expiry time, whereupon
+        /// attempts will be made to refresh the token. By default this will occur two minutes prior to the expiry of the token.
+        /// </summary>
+        /// <returns>The duration value representing the amount of time to subtract from the token expiry time.</returns>
+        public virtual TokenRefreshOptions RefreshOffset { get; } = new TokenRefreshOptions(TimeSpan.FromMinutes(2));
```

Comment:
This should be named `RefreshOptions` rather than `RefreshOffset`

Reply:
Fixed